### ISSUE TITLE
Need for a hook?

### ISSUE THREAD

**Issue by Ambrevar** [emacs-evil/evil-collection#98]
It seems like we are missing a hook for Evil Collection: if I want to override a key defined by Evil Collection in my init.el, well... I can't.  Or am I missing something?

`with-eval-after-load` won't do since Evil Collection calls the `-setup` function after it has `with-eval-after-load`ed the packages bindings.

**Comment by noctuid** [comment]
#61 adds a hook, but you could add it separately, and I could adjust the pull request.

**Comment by noctuid** [comment]
I'm waiting on a pull request I made to evil to be merged (otherwise the intercept map example in the readme won't work as expected). That doesn't matter for the translate keys code though. I could look it over and then add a disclaimer to the readme about the issue so #61 can be merged now if that would be preferable.

**Comment by jojojames** [comment]
Isn't it this one @noctuid https://github.com/emacs-evil/evil/pull/1013 ? I might be mistaken, but it looks like it's good to go?

> That doesn't matter for the translate keys code though. I could look it over and then add a disclaimer to the readme about the issue so #61 can be merged now if that would be preferable.

Works for me.

**Comment by jojojames** [comment]
Whoops accidental close, reopened.

**Comment by noctuid** [comment]
Yes, that pull request. I don't have any further changes to make to it.

**Comment by jojojames** [comment]
Looks like everyone is OK with that change? I'm wondering what's blocking it then?

**Comment by Ambrevar** [comment]
Fixed in 1853b53a8daf908244e83b8437aa2a0fb744a681.
